### ISSUE TITLE
Feature Request: Submission File Revisions

### ISSUE THREAD

**Issue by Spikestuff** [adelikat/tasvideos#1803]
Since this was accidentally "leaked" to the development team might as well just put up what we have thought of so far.

Submission file revisions where you're able to see the previous revision that was replaced by the newer one.
Sometimes this has lead to showcasing better input existing in a previous version compared to a current where sometimes splicing the older input was more beneficial.
Sometimes people just delete their userfiles so why can't we at least attempt to keep it.
Until you get someone that absolutely screws up everything is the case of no we're doing anonymity on the alias so only that changed or where it casually drops a offensive term in the previous revision.

So to summarize.

In the use case:
- Judge Errors.
- Previous revision used as a point of comparison (be able to retain information such as how many frames a file is).

Expanded idea provided by eien86 on the second point:
> Movie File 1 - Download Link - Date: XX/XX/XX - Frames 13513
> Movie File 2 - Download Link - Date: XX/XX/XX - Frames 13510 (-3,  -3 total)
> Movie File 3 - Download Link - Date: XX/XX/XX - Frames 13500 (-10, -13 total)

(Personally "Submission" instead of "Movie")
 
Ignore/Override:
- No update to input, but change to listed authors in the movie file itself.
- Casually dropping a ethnic slur was *so* 2009.

**Comment by SergioMartin86** [comment]
Adding sketch of what this might look like.
![image](https://github.com/user-attachments/assets/5f7a1130-0122-4415-8246-57766ab237fa)
